src/components/transactions/TxDetails/TxData/DecodedData/index.tsx

```diff
@@ -35,23 +46,22 @@ export const DecodedData = ({ txData, txInfo }: Props): ReactElement | null => {
   return (
     <Stack spacing={1}>
       {amountInWei !== '0' && (
-        <SendAmountBlock
-          amountInWei={amountInWei}
-          tokenInfo={{
-            type: TokenType.NATIVE_TOKEN,
-            address: ZERO_ADDRESS,
-            decimals: chainInfo?.nativeCurrency.decimals ?? 18,
-            symbol: chainInfo?.nativeCurrency.symbol ?? 'ETH',
-            logoUri: chainInfo?.nativeCurrency.logoUri,
-          }}
-        />
+        <Box pb={2}>
+          <SendAmountBlock
+            amountInWei={amountInWei}
+            tokenInfo={{
+              type: TokenType.NATIVE_TOKEN,
+              address: ZERO_ADDRESS,
+              decimals: chainInfo?.nativeCurrency.decimals ?? 18,
+              symbol: chainInfo?.nativeCurrency.symbol ?? 'ETH',
+              logoUri: chainInfo?.nativeCurrency.logoUri,
+            }}
+          />
+          <SendToBlock address={toAddress} name={name} title="Recipient" avatarSize={26} customAvatar={avatar} />
+        </Box>
       )}
-      <SendToBlock
-        address={txData.to.value}
-        title="Interact with"
-        name={isCustomTxInfo(txInfo) ? txInfo.to.name : undefined}
-        avatarSize={26}
-      />
+
+      {method && <MethodCall contractAddress={toAddress} contractName={name} contractLogo={avatar} method={method} />}
```

Comment:
In case there is an amount there is already good information visible for the user. I suggest we either hide this block in that case or move it to the top to avoid layout issues:
<img width="858" alt="Screenshot 2024-07-22 at 12 21 53" src="https://github.com/user-attachments/assets/42578c07-4004-4c2b-85d7-a0e0534165cd">

Reply:
Seeing this screenshot, similar idea like before: In case there is an address name we could use `NamedAddressInfo` inside `SendToBlock`.

Reply:
Should we also update the existing titles as seen in the screenshot above? "Send" and "Recipient" are both smaller and have a different color than the rest of them.

Reply:
@usame-algan done, I made it look like the parameters.

Reply:
In history:
<img width="938" alt="Screenshot 2024-07-26 at 10 18 19" src="https://github.com/user-attachments/assets/617302f7-3920-41cc-bd6e-2f2774b9b487">

In the tx modal:
<img width="710" alt="Screenshot 2024-07-26 at 10 18 59" src="https://github.com/user-attachments/assets/0c25ef60-81bd-4239-b969-bfda7fb58eda">

Reply:
Thanks!